stripe/test/resources/test_ephemeral_keys.py

```diff
@@ -0,0 +1,57 @@
+import stripe
+from stripe.test.helper import StripeResourceTest
+
+
+class EphemeralKeyTest(StripeResourceTest):
+
+    def test_create(self):
+        stripe.EphemeralKey.create(customer='cus_123',
+                                   api_version='2017-05-25')
+
+        self.requestor_class_mock.assert_called_with(
+            None,
+            api_version='2017-05-25',
+            account=None
+        )
+
+        self.requestor_mock.request.assert_called_with(
+            'post',
+            '/v1/ephemeral_keys',
+            {'customer': 'cus_123'},
+            None
+        )
+
+    def do_version_tests(self):
```

Comment:
Again, kind of pedantic, but do you think we could rename this to `_do_version_tests` and move it down to the bottom of the class? This will keep all the test cases together and better demonstrate that this method is supposed to be an internal helper.